packages/list-plugins/README.md

```diff
@@ -103,3 +103,20 @@ const { createdBy, updatedBy } = require('@keystonejs/list-plugins');
 ```
 
 _Note_: The API is the same.
+
+# singleton
+
+This plugin makes a list singleton by allowing only one item in the list. Useful for list which must contain only one items.
+
+## Usage
+
+```js
+const { singleton } = require('@keystonejs/list-plugins');
+
+keystone.createList('ListWithPlugin', {
+  fields: {...},
+  plugins: [
+    singleton(),
```

Comment:
This plugin does not need argument. We have this pattern to call the plugin when adding to list, if there is no argument this nesting of function call can be avoided. should this be a pattern or dependent on plugin whether to add as `[plugin]` or as `[plugin()]`.